### PR TITLE
Issue #241 PT-2 Suggestions implemented

### PR DESCRIPTION
**Pull request by Albert-Jokelin**
Issue #241

Implemented suggestions given by @nlok5923. Removed unwanted trailing spaces

### CODE REVIEW

**Comment by nlok5923** [comment]
@Albert-Jokelin please provide a static deployment link.

**Comment by Albert-Jokelin** [comment]
Hey @nlok5923! Here's the link to the netlify page: https://suspicious-clarke-a869e9.netlify.app/

File: form-correction.html [hunk 1]
```diff
@@ -960,9 +960,6 @@ <h4>Integrity Pledge
       }
     });
   </script>
-
-
```
Comment (nlok5923) [inline form-correction.html:964]: @Albert-Jokelin there are so many redundant changes like this please revert all.

**Review by nlok5923 — CHANGES_REQUESTED** [review]
Thanks PTAL

**Pull request closed without merging**